numpy/core/src/umath/loops_unary_fp.dispatch.c.src

```diff
@@ -3,19 +3,31 @@
  ** sse2 sse41
  ** vsx2
  ** neon asimd
- ** vx vxe
  **/
+
+/**
+ * We ran into lots of test failures trying to enable this file for
+ * VSE and VE on s390x (qemu) so avoiding these targets for now.
+*/
+
```

Comment:
Is is worth opening an issue for this so we can get some eyes on possible s390x performance degredation?

Reply:
We were testing to see if it was regressed either way.